server/channels/store/sqlstore/sqlx_wrapper_test.go

```diff
@@ -90,3 +90,90 @@ func TestSqlX(t *testing.T) {
 		}
 	})
 }
+
+func TestSqlxSelect(t *testing.T) {
+	t.Run("Select Methods", func(t *testing.T) {
+		testDrivers := []string{
+			model.DatabaseDriverPostgres,
+			model.DatabaseDriverMysql,
+		}
+
+		for _, driver := range testDrivers {
+			settings, err := makeSqlSettings(driver)
+			if err != nil {
+				continue
+			}
+			*settings.QueryTimeout = 1
+			store := &SqlStore{
+				rrCounter:   0,
+				srCounter:   0,
+				settings:    settings,
+				logger:      mlog.CreateConsoleTestLogger(t),
+				quitMonitor: make(chan struct{}),
+				wgMonitor:   &sync.WaitGroup{},
+			}
+
+			require.NoError(t, store.initConnection())
+			defer store.Close()
+
+			t.Run("Select with "+driver, func(t *testing.T) {
```

Comment:
These "+driver" declarations won't be needed if we have one sub-test per driver.